WordPress/Classes/Stores/ActivityStore.swift

```diff
@@ -376,6 +395,10 @@ private extension ActivityStore {
     // MARK: - Helpers
 
     func remote(site: JetpackSiteRef) -> ActivityServiceRemote? {
+        guard activityServiceRemote == nil else {
+            return activityServiceRemote!
```

Comment:
```suggestion
            return activityServiceRemote
```

Since this method returns an optional do we need to force unwrap here?

Reply:
Done